### DISCUSSION TITLE
How can I disable vertical centering?

### DISCUSSION THREAD

**Discussion by ghost** [hakimel/reveal.js#3069] [Q&A] [answered]
Sorry if the solution is already existing, but I can't seem to find anywhere.

How do I make the default start position of all content of my slides to be vertically-aligned to the top rather than the middle, **while maintaining all other default reveal.js styles**?

**Comment by hakimel** [comment, marked as answer]
You can align your content to the top by setting the `center` config option to false.

```js
Reveal.initialize({
  center: false
})
```

Here's an example with centering disabled: https://revealjs.com/demo/?center=false

**Reply by hannenz** [reply]
I am looking for how to disable vertical centering, too, and I cannot get it working in 4.2.1 as well. The slides in the example you posted above __are__ vertically centered as well -- or am I getting something really wrong here? Try resizing the viewport to some portrait-mode ratio  and it becomes very clear that the slide content is definetely centered vertically. Looking into Dev.tools confirms this: (hover over the `<div class="slides">` node). If this is the intended behavior then there is no way that the slide content starts always at the top of the viewport? Thanks for any clarification on this issue

**Reply by hakimel** [reply]
The `center` config option disables vertically centering of content *within* the slide.

The slides themselves are still scaled and centered on screen to work on all display sizes. If you want to disable this and roll your own responsive styles, try the `disableLayout` option:
```js
Reveal.initialize({ disableLayout: true })
```

Sorry about the confusion 🙃

**Reply by agneym** [reply]
@hakimel Can we disable the layout for a particular slide and keep it for the rest?
